source/_patterns/02-organisms/00-global/header.mustache

```diff
@@ -15,6 +15,7 @@
             <ul class="c-primary-nav__list" id="nav-list">
                 {{> molecules-primary-nav }}
                 {{> molecules-cart-nav }}
+                {{> molecules-country-nav }}
```

Comment:
Have you also checked to make sure that adding this on to the header doesn't break it in the tablet mode? I remember there being very little space before in that view.

Reply:
I'll check